skeleton/webpack/webpack.config.js

```diff
@@ -222,8 +222,7 @@ module.exports = ({ production } = {}, {extractCss, analyze, tests, hmr, port, h
         issuer: [{ not: [{ test: /\.html$/i }] }],
         use: extractCss ? [{
           loader: MiniCssExtractPlugin.loader
-        },
-        'css-loader'
+        }, ...cssRules
```

Comment:
This is not directly related to the changes. I have noticed that at present CSS modules have not been used. Is it better to add support for that too? If you decide to do so then, it might not make sense to import the CSS (modules) in HTML as the original local name will take no effect, as the names will be morphed (BTW which actually works vNext, with the help of convention I believe :)).

Reply:
The cssRules is 'css-loader' plus postcss. I think the previous line was missed when introducing cssRules into the code. The previous line missed postcss setup which is a bug.

Yes, there is no css-modules offered in Aurelia 1 skeletons.